client/packages/invoices/src/Prescriptions/api/operations.graphql

```diff
@@ -79,9 +79,15 @@ fragment PrescriptionRow on InvoiceNode {
 
   programId
 
-  nameInsuranceJoinId
   insuranceDiscountAmount
   insuranceDiscountPercentage
+  insurancePolicy {
+    insuranceProviders {
+      providerName
+    }
+    policyNumber
+    policyType
```

Comment:
Realised I'm not using this, maybe we shoudl either display it or remove?

Reply:
This is just referring to `policyType` right? Yea noticed it's not being used as well, wondering if we can just drop it?

Reply:
Yes Policy Type was what I was thinking of